feature-libs/checkout/base/assets/translations/en/checkout.json

```diff
@@ -23,7 +23,7 @@
     "deliveryAddressSelected": "Delivery address selected"
   },
   "checkoutMode": {
-    "deliveryMethod": "Delivery Method",
+    "deliveryMethod": "Delivery Options",
```

Comment:
Changing or removing translation keys or values is breaking change
to avoid breaking change add new translation key